src/testing/jenkins/Fossology_Build.xml

```diff
@@ -1,3 +1,7 @@
+<!-- SPDX-FileCopyrightText: © Fossology contributors
+
+     SPDX-License-Identifier: GPL-2.0-only 
+-->
```

Comment:
Just noticed a behavior with XML files that if they do not start with `<?xml version='1.0' encoding='UTF-8'?>`, their mime-type changes.

To confirm, I changed `Fossology_Build.xml` as in this PR and did not touch the `Fossology_Functional_Tests.xml`. The mime-type for file where first line was a comment changed to `text/plain` rather than `text/xml`. This will cause some serious issues down the line.

Sorry for not catching it before, but can you please make sure all xml files start with the `<?xml?>` tag and comment can come on line 2?

```
$ file -i src/testing/jenkins/Fossology_Build.xml
src/testing/jenkins/Fossology_Build.xml: text/plain; charset=utf-8
$ file -i src/testing/jenkins/Fossology_Functional_Tests.xml 
src/testing/jenkins/Fossology_Functional_Tests.xml: text/xml; charset=us-ascii
```

Reply:
Corrected.